### ISSUE TITLE
Возможно ли в рамках одного задание, рассылать сразу в два и более канала!?

### ISSUE THREAD

**Issue by BVADI** [alcortazzo/vktgbot#46]
Хочется иметь возможность, рассылать посты сразу в несколько каналов/чатов в рамках одного скрипта: 
Примерно такой конструкцией: VAR_TG_CHANNEL = '["@123456","-765233432"]'

Вообще есть кучу интересных доработок, которые работали в другом проекте (на данный момент нежив/немертв) которые хотелось бы реализовать, но к сожалению не уверен что сам осилю.
Если не трудно, выйди на связь. ТГ @BVADI

**Comment by alcortazzo** [comment]
Да, такой функционал уже давно просят. Как время появится, реализую.

**Comment by Leskov** [comment]
ERROR    | api_requests:get_video_url:52 - Error was detected when requesting data from VK: Access denied: no access to call this method - как это вылечить?

**Comment by sandboiii** [comment]
@Leskov заведи новый issue, чтобы не захламлять эту задачу

**Comment by thefoxcc** [comment]
А можно как-то обойтись без докера в dev4? или в рамках одного задание, рассылать сразу в два и более канала в v3.1
как можно исключить из парсинга ссылки такие как видео, док файлы и др. (пытался но не смог)


так же пытался сделать ограничение между отправками поста в тг но что то пошло не так (в логах ничего)
добавил в start_script `time.sleep(config.DELAY_BETWEEN_POSTS)` поставил минуту работает, поставил час прошел час след пост не опубликовалось, нажал ctrl+c и тут 2 пост прошел(сразу же опубликовалось в тг, хотя хз почему так), еще раз нажал закрылась программа.

**Comment by alcortazzo** [comment]
> А можно как-то обойтись без докера в dev4?

Можно, запускайте не через докер. Команда `python -m vktgbot`
